includes/class-sensei-utils.php

```diff
@@ -1777,16 +1777,12 @@ public static function update_course_status( $user_id, $course_id, $status = 'in
 		$comment_id = false;
 		if ( ! empty( $status ) ) {
 			$args = array(
-				'user_id'   => $user_id,
-				'post_id'   => $course_id,
-				'status'    => $status,
-				'type'      => 'sensei_course_status', /* FIELD SIZE 20 */
-				'action'    => 'update', // Update the existing status...
-				'keep_time' => true, // ...but don't change the existing timestamp
+				'user_id' => $user_id,
+				'post_id' => $course_id,
+				'status'  => $status,
+				'type'    => 'sensei_course_status', /* FIELD SIZE 20 */
+				'action'  => 'update', // Update the existing status...
 			);
-			if ( 'in-progress' == $status ) {
-				unset( $args['keep_time'] ); // Keep updating what's happened
-			}
```

Comment:
This code makes sure the comment date is updated when the status is `in-progress`. I didn't find a reason why not to do it when the status is `complete` (the only other available course status).

Reply:
I didn't find any other statuses, so it sounds reasonable.